Sources/TuistCore/Graph/GraphTraverser.swift

```diff
@@ -194,9 +194,9 @@ public class GraphTraverser: GraphTraversing {
             from: .target(name: name, path: path),
             test: { dependency in
                 isDependencyResourceBundle(dependency: dependency) && isDependencyExternal(dependency) &&
-                    canEmbedProducts(target: target)
+                    canEmbedBundles(target: target)
             },
-            skip: canDependencyEmbedProducts
+            skip: canDependencyEmbedBinaries
```

Comment:
Shouldn't this be `canDependencyEmbedBundles`?

Reply:
Yes. I updated it.